packages/website/tsconfig.json

```diff
@@ -1,12 +1,8 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "declaration": true,
-    "emitDeclarationOnly": true,
     "rootDir": "./src",
-    "outDir": "./dist",
-    "tsBuildInfoFile": "./dist/.tsbuildinfo",
-    "lib": ["ESNext", "DOM", "DOM.Iterable"]
+    "outDir": "./dist"
```

Comment:
It feels good to me, but let me mention @himself65 to check if this can cause any problems.

Reply:
I think we should keep `tsBuildInfoFile` otherwise it will generate to the package dir

Reply:
Like this 
<img width="493" alt="image" src="https://github.com/dai-shi/waku/assets/14026360/b1b280f2-cb51-445d-88fb-defd9750cb1e">

Reply:
I see. Is it for incremental build? I'd put it outside `dist`. Let me try.

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html

We could put it anywhere. It's just a cache file for incremental build.

Reply:
Yeah, I don't think it's necessary in dist: https://unpkg.com/browse/waku@0.16.0/dist/.tsbuildinfo